doc/source/configuration/wazuh.rst

```diff
@@ -73,7 +73,7 @@ Optional: define LVM volumes ``etc/kayobe/inventory/group_vars/wazuh-manager/lvm
           size: "100%VG"
           filesystem: "ext4"
           mount: true
-          mntp: “/var/lib/elasticsearch”
+          mntp: "/var/lib/elasticsearch"
```

Comment:
@jackhodgkiss actually ended up mounting `/var/ossec`, claiming it has greater storage requirements. Perhaps we actually need both?